vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/dataview/GridListDataViewTest.java

```diff
@@ -71,8 +74,8 @@ public void dataViewWithoutItems_exceptionThrown() {
     public void dataViewWithItems_returnsExpectedItemsForMethods() {
         String[] items = new String[] { "item1", "item2", "item3", "item4" };
         Grid<String> grid = new Grid<>();
-        grid.setDataProvider(DataProvider.ofItems(items));
-        GridListDataView<String> dataView = new GridListDataView<>(grid);
+        GridListDataView<String> dataView = grid
+                .setDataProvider(DataProvider.ofItems(items));
```

Comment:
Why does this test use `DataProvider.ofItems(items` instead of just `setDataProvider(items)` ???

Reply:
Might have been some specific test thing at some point and was just left.
Done.